docs/accessibility/get-started/introduction.mdx

```diff
@@ -44,34 +44,104 @@ From there, you can integrate with CI to set your own standards for handling the
 
 [Get started with Cypress Accessibility ➜](/accessibility/get-started/setup)
 
-## How it Works
-
-Cypress Accessibility provides organized, visual, and actionable accessibility reports, based completely on the tests you record to Cypress Cloud, and powered by Axe Core® by Deque Systems. This is the documentation page for that product. To learn about the general topic of accessibility testing using the Cypress App, see [our guide on this topic](/app/guides/accessibility-testing).
-
-Cypress Accessibility generates a sortable, filterable report, to see scores and violation counts by page/component, or a combined rule-based report for the entire run. It also includes a drill-in view with live DOM snapshots highlighting each issue detected.
-
-:::caution
-
-Cypress Accessibility generates reports using [Cypress Test Replay](/cloud/features/test-replay) data and requires Cypress v13+.
+## Top Guides
+
+<ul class="guidesList">
+  <li class="card">
+    <a href="/accessibility/guides/improve-accessibility">
+      <Icon name="chart-line" />
+      <h3>Improve Accessibility</h3>
```

Comment:
If we want to keep this structure, we can shorten the name of the link by referencing the card titles as the label. This way of labelling means there's just one source of truth for the proper name of the card & not two things to update if we re-order the content or whatever.

```suggestion
    <a href="/accessibility/guides/improve-accessibility" aria-labelledby="card-title-1">
      <Icon name="chart-line" />
      <h3 id="card-title-1">Improve Accessibility</h3>
```